### DISCUSSION TITLE
Setup OpenCollective to gain some payback for the effort

### DISCUSSION THREAD

**Discussion by Shinigami92** [lodash/lodash#5723] [Ideas]
Setup an https://opencollective.com/ account so the community can pay back some money for the effort

This money can not only be used to compensate the work of the core maintainers, but also to e.g. pay contributors of any kind
Even small amounts like $10 for implementing a big feature can help a bit

---

sadly it looks like https://opencollective.com/lodash is already claimed by someone referring to a non-existing [lodah](https://github.com/lodah) **wihtout** s, so it might be needed to specially contact open collective support or just take a different name

**Comment by jdalton** [comment]
Thanks @Shinigami92. I've reached out to their support to claim the squatted name.
